app/helpers/application_helper/toolbar/firmware_registries_center.rb

```diff
@@ -7,7 +7,7 @@ class ApplicationHelper::Toolbar::FirmwareRegistriesCenter < ApplicationHelper::
         'fa fa-refresh fa-lg',
         N_('Refresh this page'),
         N_('Refresh'),
-        :data => { 'function' => '() => window.location.reload()' }
+        :data => { 'function' => 'window.location.reload' }
```

Comment:
```suggestion
        :data => { 'function' => 'function() { window.location.reload() }' }
```

Passing the function directly crashes for me (see the other comment), I suggest we keep it in lambda, but do it old-style. I'm all ears as to why this works when in lambda, I don't see how it could change anything (but it does)? :)